src/gc/windows/gcenv.windows.cpp

```diff
@@ -484,6 +484,12 @@ void GCToOSInterface::GetMemoryStatus(uint32_t* memory_load, uint64_t* available
         *available_page_file = ms.ullAvailPageFile;
 }
 
+// Get size of an OS memory page
+void GCToOSInterface::GetPageSize()
+{
+    return 4096;
```

Comment:
would be nice to assert that `g_SystemInfo.dwPageSize` equals 4096 here, our handling of OS write watch is going to be subtly wrong otherwise.